app/views/users/sessions.py

```diff
@@ -141,6 +144,95 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/', methods=('GET', 'POST'))
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    data = "Speaker Name, Speaker Session(s) , Speaker Email, Speaker Status," \
+           " Speaker Mobile, Speaker Organisation, Speaker Position \n"
+
+    for session in sessions:
+        speakers = session.speakers
+        for speaker in speakers:
+            try:
+                data += speaker.name+','
+            except:
```

Comment:
Applicable for all other try-except block below. Please try using try-except block for exception handling rather than to check whether an error in condition.